automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -59,6 +59,7 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
   lazy val ronEmail = ronCreds.email
 
   val clusterPatience = PatienceConfig(timeout = scaled(Span(10, Minutes)), interval = scaled(Span(20, Seconds)))
+  val clusterStopAfterCreatePatience = PatienceConfig(timeout = scaled(Span(20, Minutes)), interval = scaled(Span(20, Seconds)))
```

Comment:
Adding longer patience for stop-after-creation. See https://broadworkbench.atlassian.net/browse/IA-1251

Reply:
Stopping a cluster taking longer than 10 mins sounds awfully long but I guess  🤷‍♂

Reply:
In this instance creating it took 5mins and stopping it took 5 mins.

Reply:
Oh the timeout is for both creating and stopping. That feels better :)